demos/demo_camera.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import time
+import numpy as np
+import pickle
+import cv2
+
+import robot_interfaces
+
+camera_data = robot_interfaces.camera.SensorData()
+camera_handle = robot_interfaces.camera.OpenCVDriver()
+camera_backend = robot_interfaces.camera.SensorBackend(camera_handle, camera_data)
+camera_frontend = robot_interfaces.camera.SensorFrontend(camera_data)
+
+CameraObservation = robot_interfaces.camera.OpenCVObservation
+
+num_imgs = 100000000
+img_list = []
+
+for i in range(num_imgs):
+    OpenCVObservation = (camera_frontend.get_latest_observation())
+    img_list.append(OpenCVObservation)
+    print("At image", i)
+    window_name = 'image'
+    cv2.imshow(window_name, np.array(OpenCVObservation.image))
```

Comment:
The blog post mentioned it's a conversion to np.ndarray but it doesn't seem like that to me since I needed to explicitly convert is to an np array. And it outputs the type as the name that's passes to pybind11::class_<cv::Mat>.

Reply:
At the bottom of the blog he actually also converts to `np.array` manually.  However, he adds a `copy=False` argument to it, to avoid copy of the data:
```
def get_image():
    im = myextension.get_image()
    return np.array(im, copy=False)
```
From what I understood `np.asarray(im)` should be the same as `np.array(im, copy=False)`, so I would suggest to use that in our code.